services-directions-models/src/test/java/com/mapbox/api/directions/v5/models/IncidentTest.java

```diff
@@ -17,31 +17,21 @@ public void sanity(){
 
   @Test
   public void testSerializableObject() throws Exception {
-    Incident incident = Incident.builder()
-      .id("some_id")
-      .alertcCodes(Lists.newArrayList(431, 2123, 934))
-      .closed(true)
-      .congestion(Congestion.builder().value(10).build())
-      .creationTime("2020-11-18T11:34:14Z")
-      .startTime("2021-11-18T11:32:14Z")
-      .endTime("2021-11-18T12:12:08Z")
-      .description("description")
-      .longDescription("long description")
-      .geometryIndexStart(1)
-      .geometryIndexEnd(943)
-      .impact(Incident.IMPACT_MAJOR)
-      .subType("sub type")
-      .subTypeDescription("sub type desc")
-      .type(Incident.INCIDENT_DISABLED_VEHICLE)
-      .lanesBlocked(Lists.<String>newArrayList())
-      .numLanesBlocked(null)
-      .countryCodeAlpha2("US")
-      .countryCodeAlpha3("USA")
-      .build();
+    Incident incident = getFilledIncident();
     byte[] serialized = TestUtils.serialize(incident);
     assertEquals(incident, deserialize(serialized, Incident.class));
   }
 
+  @Test
+  public void testToAndFromJson() {
```

Comment:
We should also remove `test` prefix from tests names, it's implicit with the usage of `@Test` annotation.

Same thing applies to other tests above and below.

Reply:
I don't see anything bad in having a `test` word in the beginning in a test name. But if you have strong opinion about this, I don't mind changing the names.
Updated.